project-2/Handling_Missing_Data.ipynb

```diff
@@ -0,0 +1,8520 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # start = data_issued['Issued Date'].max()
</pre><p>Do we need to see this comment or it's just for your own use? If it's only for you, it shouldn't be in the committed submission</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Data-Epic/Humphery_Ufuoma/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>